README.md

```diff
@@ -58,6 +58,10 @@ Optionals are the simplest type of property that can be made `Resilient`. A prop
 
 `Resilient` can also be applied to an array or an optional array (`[T]?`). A property written as `@Resilient var foo: [Int]` will be initialized with an empty array if the `foo` key is missing or if the value is something unexpected, like `String`. Likewise, if any _element_ of this array fails to decode, that element will be omitted. The optional array variant of this will set the value to `nil` if the key is missing or has a null value, and an empty array otherwise.
 
+### `Dictionary`
+
+`Resilient` can also be applied to a (string-keyed) dictionary or an optional dictionary (`[String: T]?`). A property written as `@Resilient var foo: [String: Int]` will be initialized with an empty dictionary if the `foo` key is missing or if the value is something unexpected, like `String`. Likewise, if any _value_ in the dictionary fails to decode, that value will be omitted. The optional dictionary variant of this will set the value to `nil` if the key is missing or has a null value, and an empty array otherwise.
```

Comment:
I think we should say something here about how we ignore fallbacks for raw representables in dictionaries.
Do we have the same behavior for raw representables in arrays?

Reply:
I added a comment to the `ResilientRawRepresentable` section, since they aren't yet introduced at this point in the doc.